src/MCPServer/share/mysql_dev_9647_archivesspace_inherit_notes.sql

```diff
@@ -0,0 +1,5 @@
+-- Add column inherit_notes column to administration_archivesspaceconfig
+ALTER TABLE `administration_archivesspaceconfig` ADD COLUMN `inherit_notes` bool NOT NULL AFTER `repository`;
```

Comment:
We haven't really. :sweat_smile: Previously all tables were created by SQL. We recently started adding tables created by Django, but I think this might be the first case where we had to modify one (so syncdb created something different than an upgrade produced).